.ci/pipeline/nightly.yml

```diff
@@ -34,7 +34,7 @@ variables:
 jobs:
 - job: Coverity
   pool:
-    vmImage: 'ubuntu-latest'
+    vmImage: 'ubuntu-22.04'
```

Comment:
This should be in a separate PR.  Forcing the VM OS for Coverity/Jupyter checks doesn't seem related and isn't mentioned in the description. Doesn't seem related to the conda build. (Will fast track review when split off).

Reply:
It is actually mentioned: `- Pin CI images versions`.
As CI has update in this PR anyway, I thought it's convenient to include pinning as no actual change in used images is happening.

Reply:
I went on ahead and changed the title to make sure this was also there, I think splitting off to a separate PR is a waste of time, just as long as it doesn't get lost in the shuffle.